content/posts/2025-02-26-whats_new_1_8/index.md

```diff
@@ -0,0 +1,319 @@
+tags: network,observability,new,xlate,OVN,UDN,eBPF
+authors: [stleerh]
+rhDevBlogURL:
+---
+
+_Thanks to Joel Takvorian, Mohamed S. Mahmoud, and Julien Pinsonneau for reviewing._
+
+Network observability is one of the most critical and must-have components in your Kubernetes cluster.  When networking stops working or slows down, your productivity is essentially grinded to a halt.  With that said, let me introduce you to Network Observability 1.8, which aligns with Red Hat [OpenShift Container Platform (OCP) 4.18](https://docs.openshift.com/container-platform/4.18/release_notes/ocp-4-18-release-notes.html).  While it supports older versions of OCP and any Kubernetes cluster, many of the new features require 4.18 and specifically, OVN-Kubernetes as your Container Network Interface (CNI).
+
+If you read [my blogs](https://developers.redhat.com/author/steven-lee) on this topic, I tend to give a sneak peek on Developer Preview (DP) features as well as Technology Preview (TP) features.  In both cases, they should not be used in production until they reach General Availability (GA).  I also cover only the new features, so if you want an overview of everything in Network Observability, check out [About Network Observability](https://docs.openshift.com/container-platform/4.18/observability/network_observability/network-observability-overview.html) in the Red Hat documentation.
+
+## Setup
+
+To begin, you should have an OCP cluster, preferably 4.18 so you can test the new features, and [Network Observability Operator 1.8 installed](https://docs.openshift.com/container-platform/4.18/observability/network_observability/installing-operators.html#network-observability-operator-installation_network_observability).  Install Loki using the Loki Operator or  a monolithic Loki (non-production) by following the instructions in the Network Observability Operator in the OpenShift web console.  Create a FlowCollector instance and enable Loki.  By default, it should be running OVN-Kubernetes.  You should also have the OpenShift CLI ([`oc`](https://docs.openshift.com/container-platform/4.18/cli_reference/openshift_cli/getting-started-cli.html)) installed on your computer.  Now let's dive in!
+
+## Features
+
+This release is unusual in that there are just two items I will cover in the GA category.  They are packet translation and eBPF resource reduction.  The rest are the upcoming features that fall into the Developer Preview and Technology Preview categories.
+
+There were also significant changes in the Network Observability CLI, a kubectl plug-in that lets you use Network Observability from the command line, separate from the operator.  I've talked about this in the past, but it's gotten a lot bigger, so you can now read all about it in [Network Observability On Demand 1.8 Update](https://netobserv.io/posts/network-observability-on-demand-1-8-update/).
```

Comment:
Instead of  "it's gotten a lot bigger", maybe "it has many great new features"

Reply:
reworded